src/Plugin/resource/DataProvider/CachedDataProvider.php

```diff
@@ -261,10 +257,33 @@ public static function invalidateEntityCache($cid) {
           // in user.
           $uid = $GLOBALS['user']->uid;
         }
-        $version_cid = 'v' . $version['major'] . '.' . $version['minor'] . '::' . $plugin->getResourceName() . '::uu' . $uid;
-        $plugin->cacheInvalidate($version_cid . '::' . $cid);
+        $version_cid = 'v' . $version['major'] . '.' . $version['minor'] . '::' . $plugin->getResourceMachineName() . '::uu' . $uid;
+        $plugin->getDataProvider()->cacheInvalidate($version_cid . '::' . $cid);
+      }
+    }
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function cacheInvalidate($cid) {
+    $options = $this->getOptions();
+    $cache_info = $options['renderCache'];
+
+    if (!$cache_info['simple_invalidate']) {
+      // Simple invalidation is disabled. This means it is up to the
+      // implementing module to take care of the invalidation.
+      return;
+    }
+    // If the $cid is not '*' then remove the asterisk since it can mess with
+    // dynamically built wildcards.
```

Comment:
This was copied an pasted from old code. This refers to standard `cache_clear` https://api.drupal.org/api/drupal/includes%21cache.inc/function/DrupalCacheInterface%3A%3Aclear/7